internal/service/alarms/internal/alarms_server.go

```diff
@@ -57,12 +60,10 @@ func (a *AlarmsServer) GetAlarms(ctx context.Context, request api.GetAlarmsReque
 // GetAlarm returns an AlarmEventRecord with a given ID
 func (a *AlarmsServer) GetAlarm(ctx context.Context, request api.GetAlarmRequestObject) (api.GetAlarmResponseObject, error) {
 	aerModel, err := a.AlarmsRepository.GetAlarmEventRecordWithUuid(ctx, request.AlarmEventRecordId)
-	if err != nil {
```

Comment:
I think it is more natural to check first if the error is different than nil and then check if it is a specific error

Reply:
Yeah, I've had discussions about this in the past. It's a matter of personal taste I would say. Personally, I like to check nil first and then the specific error inside. This is fine as well of  course(although, I've usually seen it without the else because it doesn't add anything, just if != nil below).